maestro-cli/src/main/java/maestro/cli/runner/TestRunner.kt

```diff
@@ -0,0 +1,77 @@
+package maestro.cli.runner
+
+import maestro.Maestro
+import maestro.orchestra.NoInputException
+import maestro.orchestra.SyntaxError
+import maestro.orchestra.yaml.YamlCommandReader
+import org.fusesource.jansi.Ansi
+import org.fusesource.jansi.AnsiConsole
+import java.io.File
+import kotlin.concurrent.thread
+
+object TestRunner {
+
+    fun run(
+        maestro: Maestro,
+        flowFile: File,
+        continuous: Boolean,
+    ): Int {
+        AnsiConsole.systemInstall()
+        println(Ansi.ansi().eraseScreen())
+
+        val view = ResultView()
+
+        fun runFlow(): Int {
+            val result = runCatching(view) {
+                MaestroCommandRunner.run(maestro, view, flowFile)
+            }
+            return if (result == true) 0 else 1
+        }
+
+        if (!continuous) {
```

Comment:
I would've preferred to have single and continuous runners in separate files as they have a fair number of differences. I.e. in Single runner, we would actually not need to erase the screen as it will be used for running in CI (so the `view` might end up being different)

Reply:
Split into two separate methods